.gitignore

```diff
@@ -127,3 +127,8 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.idea
+/testproject/slave.sqlite3
```

Comment:
Let's respect west collegues and use primary-replica terminology.

Reply:
fix